src/core/mainloop/connection.c

```diff
@@ -5387,6 +5387,7 @@ get_proxy_addrport(tor_addr_t *addr, uint16_t *port, int *proxy_type,
       tor_addr_copy(addr, &transport->addr);
       *port = transport->port;
       *proxy_type = transport->socks_version;
+      *is_pt = 1;
```

Comment:
Does anything set *is_pt to 0 when we are _not_ a PT?  It would make sense to do that IMO.

Reply:
I think this happens downstream (in the `or_connection_t`) as a side effect of `tor_alloc_zero()`, which I think we rely on for the other flag bits in that structure. I can add an explicit setting of zero for robustness.